config/prometheus/role.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  labels:
+    app.kubernetes.io/name: role
+    app.kubernetes.io/component: metrics
+    app.kubernetes.io/part-of: lightspeed-operator
+  name: prometheus-k8s
+  namespace: openshift-lightspeed
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - services
+      - endpoints
+      - pods
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
```

Comment:
@raptorsun do we need the ingresses for us ?

Reply:
Prometheus will look for ingresses, too. That is not neccessary for OLS stack but it can avoid error logs in Prometheus.